cmd/internal/cli/build.go

```diff
@@ -28,24 +28,28 @@ import (
 )
 
 var buildArgs struct {
-	sections      []string
-	bindPaths     []string
-	mounts        []string
-	libraryURL    string
-	keyServerURL  string
-	webURL        string
-	encrypt       bool
-	fakeroot      bool
-	fixPerms      bool
-	isJSON        bool
-	noCleanUp     bool
-	noTest        bool
-	sandbox       bool
-	update        bool
-	nvidia        bool
-	nvccli        bool
-	rocm          bool
-	writableTmpfs bool // For test section only
+	sections       []string
+	bindPaths      []string
+	mounts         []string
+	libraryURL     string
+	keyServerURL   string
+	webURL         string
+	encrypt        bool
+	fakeroot       bool
+	fixPerms       bool
+	isJSON         bool
+	noCleanUp      bool
+	noTest         bool
+	sandbox        bool
+	update         bool
+	nvidia         bool
+	nvccli         bool
+	rocm           bool
+	writableTmpfs  bool // For test section only
+	userns         bool // enabling userns flag
+	ignoreSubuid   bool // ignoring subuid entires flag (hidden)
+	ignoreFakeroot bool // ignoring fakeroot command (hidden)
```

Comment:
I think `ignoreFakeroot` is a misleading name.  I suggest `ignoreFakerootCmd` (using Cmd instead of Command to keep it from getting too long).

better comment: `ignore fakeroot commnd (hidden)`